index.html

```diff
@@ -40,6 +40,7 @@
                     </ul>
                     <span> VE Campaign</span>
                 </div>
+                <a class="logo" href="index.html"><img src="images/logo.png" alt="logo"> </a> 
             </nav>
             <div class="sub-container">
```

Comment:
Please be sure to address this requested changes to your main section so that your design looks almost exactly to the template 😁 Try giving your elements some margins and you could increase the height as well 📝

Reply:
some other reviewer said its not necessary

Reply:
- [ ] Kindly as requested by the first and second reviewer [here](https://github.com/Emmykage/Capstone-Project/pull/1#discussion_r877617270) make sure to address the requested changes :wink: 💯 

- Here is the color you are supposed to use:

![color](https://user-images.githubusercontent.com/79058364/169443931-2f7f034a-dae3-4831-a82b-4be59524c4c8.png)